### PR TITLE
Update tippy.js dependency

### PR DESCRIPTION
**Pull request by nicolethoen**
Addresses: [#3252 ](https://github.com/patternfly/patternfly-react/issues/3252)

### CODE REVIEW

File: packages/patternfly-4/react-core/src/components/Popover/Popover.tsx [hunk 1]
```diff
@@ -95,7 +95,7 @@ export interface PopoverProps {
   /** z-index of the popover */
   zIndex?: number;
   /** additional Props to pass through to tippy.js */
-  tippyProps?: TippyProps;
+  tippyProps?: object;
```
Comment (redallen) [inline packages/patternfly-4/react-core/src/components/Popover/Popover.tsx]: Are Tippy's types no longer working? Avoiding `object` would be nice for consumers.

Comment (nicolethoen) [reply packages/patternfly-4/react-core/src/components/Popover/Popover.tsx]: there are like 43 required TippyProps. So in the tests, for example, we tried only passing two tippy props, we get a type error.

Comment (redallen) [reply packages/patternfly-4/react-core/src/components/Popover/Popover.tsx]: Does `Partial<TippyProps>` work?

**Comment by redallen** [comment]
Thanks Nicole!

**Pull request merged**